security/ssh_certificate.ksy

```diff
@@ -0,0 +1,159 @@
+meta:
+  id: ssh_certificate
+  title: OpenSSH Certificate
+  license: CC0-1.0
+  imports:
+    - /security/ssh_public_key
+  endian: be
+doc: |
+  OpenSSH Certificates are simple certs used by OpenSSH.
+doc-ref: 'https://cvsweb.openbsd.org/cgi-bin/cvsweb/~checkout~/src/usr.bin/ssh/PROTOCOL.certkeys?rev=1.17&content-type=text/plain'
+seq:
+  - id: cert_type
+    doc: type of ssh certificate
+    type: cstring_utf8
```

Comment:
For all `seq`uence attributes and `instances`, the `doc` key should go **below** all other keys, see https://doc.kaitai.io/ksy_style_guide.html#seq-attr.